scripts/functions.sh

```diff
@@ -394,22 +394,21 @@ function setup_logging() {
 }
 
 function geoserver_logging() {
-    if [[ ${CLUSTERING} =~ [Tt][Rr][Uu][Ee] ]]; then
-        export LOG_PATH=${CLUSTER_CONFIG_DIR}/geoserver-${HOSTNAME}.log
-      else
-        create_dir "${GEOSERVER_DATA_DIR}"/logs
-        export LOG_PATH=${GEOSERVER_DATA_DIR}/logs/geoserver-${HOSTNAME}.log
-    fi
+  if [[ ${CLUSTERING} =~ [Tt][Rr][Uu][Ee] ]]; then
+      export LOG_PATH=${CLUSTER_CONFIG_DIR}/geoserver-${HOSTNAME}.log
+    else
+      create_dir "${GEOSERVER_LOG_DIR}"
```

Comment:
We need to add this env variable to the emtrypoint logic where we chown for the directory to be owned by the specific user

Reply:
Updated the existing chown to use GEOSERVER_LOG_DIR instead of GEOSERVER_DATA_DIR/logs